en/components/combo.md

```diff
@@ -106,6 +106,76 @@ When igxCombo is opened, allow custom values are enabled and add item button is
 
 - `ArrowUp` focus will be moved back to the last list item or if list is empty will be moved to the search input.
 
+## Styling
+Using the [Ignite UI for Angular Theming](../themes/index.md), we can greatly alter the **igx-combo** appearance. 
```

Comment:
Theme files are in the same relative directory, so it would be `themes/index.md` instead of `../themes/index.md`

Reply:
Relevant for all other links as well